openapi/crm/crm.json

```diff
@@ -0,0 +1,319 @@
+{
+  "openapi": "3.1.0",
+  "x-stoplight": {
+    "id": "iyzxipdhj3szx"
+  },
+  "info": {
+    "title": "crm",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "https://prod.apigateway.co/org",
+      "description": "Production"
+    },
+    {
+      "url": "https://demo.apigateway.co/org",
+      "description": "Demo"
+    },
+    {
+      "url": "http://localhost:11001/org",
+      "description": "Local"
+    },
+    {
+      "url": "{local}/org",
+      "description": "Local"
+    }
+  ],
+  "paths": {
+    "/{namespace}/{resourceTypeCode}/meta/fields": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        }
+      ],
+      "get": {
+        "summary": "Get field schema",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "This API follows [Open API schema](https://spec.openapis.org/oas/v3.1.0#schema-object) of structure for response.",
+            "content": {
+              "application/json": {
+                "examples": {
+                  "contacts Resource": {
+                    "value": {
+                      "CRMContact": {
+                        "title": "CRMContact",
```

Comment:
ListFieldSchema basically provides all the attributes for the object that defines the Contact for the given PID/AGID. The response from api-gateway will as below for the endpoint `/org/{namespace}/{resourceTypeCode}/meta/fields`
`{
    "type": "object",
    "description": "CRM Field Schema of resourceType Contact for the partner VA",
    "properties": {
        "platform__contact_additional_salesperson_ids": {
            "description": "Identifiers for additional salespeople in the platform.",
            "title": "Additional Salesperson IDs",
            "type": "11"
        },...
}`
Based on the details defined,`CRMContact` schema displayed is only to provide the reader a general idea of the attributes for the CRM Contact can/will have without divulging in the details of crm schema for the Contact?

Reply:
Yes, correct.